service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java

```diff
@@ -13,14 +13,15 @@
     private final LDClient ldClient;
     private final LDUser ldUser;
     private final LDUser.Builder ldUserBuilder;
+    private final String ldUserKey;
 
     @Autowired
     public FeatureToggleService(LDClient ldClient, @Value("${ld.user_key}") String ldUserKey) {
         this.ldClient = ldClient;
         this.ldUserBuilder = new LDUser.Builder(ldUserKey)
             .custom("timestamp", String.valueOf(System.currentTimeMillis()));
```

Comment:
Please fix other instances of the problem at the same time - tthis timestamp is broken - it would always send data when the userBuilder was created which seems wrong (it would work if the timestamp would evaluate in lambda)

Notification for judge seems to be broken as well, as Tomek suggested the builder should be remove dfrom class, it's bad design.

Additionoally, please prove in the tests.